autoload/limelight.vim

```diff
@@ -138,7 +138,7 @@ function! s:dim(coeff)
   let synid = synIDtrans(hlID('Normal'))
   let fg = synIDattr(synid, 'fg#')
   let bg = synIDattr(synid, 'bg#')
-  let bg = bg == 'none' ? 0 : bg
+  let bg = bg == 'none' ? bg : 0
```

Comment:
This change doesn't make sense to me. Why are we discarding a valid`bg` value and only keeping `none`?